### DISCUSSION TITLE
INACTIVE keyword in Tinker9

### DISCUSSION THREAD

**Discussion by generalig** [TinkerTools/tinker-gpu#254] [Q&A]
Hi all. I am looking for a MD code that can run in GPUs, with frozen atoms; that could be achieved in Tinker8 with the INACTIVE keyword, but I am not sure that has been ported to the GPU version. It is not listed in the keyword section of the manual, so I imagine it has not, but would like to double check, as I have used Tinker many years ago, and remember it as a great piece of software. I would appreciate anyone who knows about this. 
Thanks, 
Ignacio

**Comment by jayponder** [comment]
Unfortunately, I think you are correct. The INACTIVE keyword is not implemented in the GPU code. Someone who has done development on Tinker9 more recently correct me if this is wrong! There are lots of options available in the CPU Tinker8 code that are not in the GPU Tinker9, generally to maintain efficiency in the GPU code.

**Reply by generalig** [reply]
I see. Thanks for you reply!
